homeassistant/components/brother/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Brother Printer",
   "documentation": "https://www.home-assistant.io/integrations/brother",
   "codeowners": ["@bieniu"],
-  "requirements": ["brother==1.1.0"],
+  "requirements": ["brother==1.2.2"],
```

Comment:
We don't want libraries to pin their requirements to exact versions. Please be as forgiving as possible when pinning requirements.

Normally the production requirements specified in `setup.py` are left with a loose pinning and the development requirements in `requirements.txt` are pinned to an exact version.

https://github.com/bieniu/brother/blob/b339ba3b4358ea2f158aff438bb1f243f26c8ae4/setup.py#L26

Reply:
Ok, I'll unpin `pysnmplib` version.

Reply:
Done